src/www/ui/api/Controllers/GroupController.php

```diff
@@ -92,4 +92,63 @@ public function createGroup($request, $response, $args)
     }
     return $response->withJson($returnVal->getArray(), $returnVal->getCode());
   }
+
+  /**
+   * Get users with their roles from a given group
+   *
+   * @param ServerRequestInterface $request
+   * @param ResponseHelper $response
+   * @param array $args
+   * @return ResponseHelper
+   */
+  public function getGroupMembers($request, $response, $args)
+  {
+    $userId = $this->restHelper->getUserId();
+    /** @var UserDao */
+    $userDao = $this->restHelper->getUserDao();
+    $groupMap = $userDao->getAdminGroupMap($userId, $_SESSION[Auth::USER_LEVEL]);
+    $res = null;
+
+    if (empty($groupMap)) {
+      $res = new Info(400, "You have no permission to manage any group.", InfoType::ERROR);
+    } else {
+      $groupId = null;
+
+      /**
+       * Let group be the default one , if there's no group provided in the url params. *
+       **/
```

Comment:
Not sure that will be a valid use case.
It can, however, be resolved to currently active group id (`$this->restHelper->getGroupId()`).

But, with the suggested changes in the URL format, empty id will result in a 404/400 response.